### PR TITLE
Fix the displayed expression for string assertion `IsEqualTo`

### PR DESCRIPTION
**Pull request by vbreuss**
The displayed expression for the string assertion `IsEqualTo` incorrectly also adds the default `StringComparison.Ordinal`, although it is not provided.

### CODE REVIEW

File: TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs [hunk 1]
```diff
@@ -13,7 +13,10 @@ public static class StringIsExtensions
 {
     public static StringEqualToAssertionBuilderWrapper IsEqualTo(this IValueSource<string> valueSource, string expected, [CallerArgumentExpression("expected")] string doNotPopulateThisValue1 = "")
     {
-        return IsEqualTo(valueSource, expected, StringComparison.Ordinal, doNotPopulateThisValue1);
+        var assertionBuilder = valueSource.RegisterAssertion(new StringEqualsExpectedValueAssertCondition(expected, StringComparison.Ordinal)
```
Comment (thomhurst) [inline TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs]: Could we still call the overload but pass in an empty expression?

**Comment by thomhurst** [comment]
Got a merge conflict

**Comment by vbreuss** [comment]
> Got a merge conflict 

I will have a look at it later...

File: TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs (hunk 1 above)
Comment (vbreuss) [reply TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs]: I tried it, but both, with null or an empty string, the result contained it in the generated expression string, so I had to revert this commit.

Comment (thomhurst) [reply TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs]: Can you try again with the latest main merged in? I believe I've sorted that

Comment (vbreuss) [reply TUnit.Assertions/Extensions/Is/Strings/StringIsExtensions.cs]: I merged in `main` and it seems to work when I provide `null`.

**Pull request merged**